src/app/Http/Controllers/Admin/UniquePageCrudController.php

```diff
@@ -0,0 +1,245 @@
+<?php
+
+namespace Backpack\PageManager\app\Http\Controllers\Admin;
+
+use App\UniquePages;
+use Illuminate\Http\Request;
+use Backpack\PageManager\app\TraitReflections;
+use Backpack\CRUD\app\Http\Controllers\CrudController;
+use Backpack\CRUD\app\Http\Controllers\CrudFeatures\SaveActions;
+
+class UniquePageCrudController extends CrudController
+{
+    use SaveActions;
+    use UniquePages;
+    use TraitReflections;
+
+    public function setup()
+    {
+        parent::__construct();
```

Comment:
Why is the **constructor** called from setup?

Reply:
Good point... this was just a copy&paste from the original controller...